yun/libraries/Parse/src/internal/ParseClient.h

```diff
@@ -127,13 +127,14 @@ class ParseClient {
    *  \param   httpVerb - GET/DELETE/PUT/POST
    *  \param   httpPath - the endpoint of REST API e.g. /1/installations
    *  \param   requestBody - http request body in json format, leave it as "" for DELTE/GET requests
-   *  \param   urlParams - leave it as "" unless to perform a Parse query,
-   *            use it to specify query condition e.g. where={"KEY1"":VALUE1}&limit=10&keys=KEY1,KEY2
-   *  \result response of request
+   *           or use it to specify query condition e.g. where={"KEY1"":VALUE1}&limit=10&keys=KEY1,KEY2
+   *           NOTE: isQuery needs to be set as true if you use requestBody as query condition
+   *  \param   isQuery - if the request is a query
+   *  \result  response of request
    *  NOTE: ParseObjectCreate, ParseObjectUpdate, ParseObjectDelete, ParseGETObject,
    *  ParseQuery, ParseCloudFunction, ParseTrackEvent can also be used for REST API call
    */
-  ParseResponse sendRequest(const char* httpVerb, const char* httpPath, const char* requestBody, const char* urlParams);
+  ParseResponse sendRequest(const char* httpVerb, const char* httpPath, const char* requestBody, bool isQuery);
```

Comment:
@stanleyw  
Without explicit indication, when both "requestBody" and "urlParams" are set as empty strings(""), we are not able to tell if it is a query or a GET request.  We have different handling strategy for return results of query and GET request. e.g.  GET /1/classes/TestObject is a query, it returns a json array which needed to be iterated by user. While we take it as a GET request, which directly returns the whole return results.